packages/core-components/src/components/AlertDisplay/AlertDisplay.tsx

```diff
@@ -121,10 +123,12 @@ export function AlertDisplay(props: AlertDisplayProps) {
         <Typography component="span">
           {String(firstMessage.message)}
           {messages.length > 1 && (
-            <em>{` (${messages.length - 1} older ${pluralize(
-              'message',
-              messages.length - 1,
-            )})`}</em>
+            <em>
+              {t('alertDisplay.message', {
+                num: String(messages.length - 1),
```

Comment:
Im pretty sure that in the translation string you should be able to do `{{ count }}` instead of having two variables for this. Please let us know if this doesn't work :)

Reply:
done